docs/configuration.md

```diff
@@ -30,6 +30,9 @@
   are both on the same file (count, default 0 (no backpressure))
   e.g. backpressure = 10 [writer = 100.log, slowest reader = 89.log, delta = 11]
 * **hostname** - hostname used in logging/messages (default gethostname())
+* **remove_checkpoint_on_stop** - wether plugin checkpoint should be
```

Comment:
typo: whether
but could probably be rephrased anyway:
`removes the checkpoint entry when the plugin is stopped, terminated, or has been manually removed (default false)`